docker.go

```diff
@@ -76,6 +76,7 @@ func (h *handler) startContainer(ctrInfo containerInfo) (string, *websocket.Conn
 	r, err := h.dcli.ContainerCreate(
 		context.Background(),
 		&container.Config{
+			User:         "nobody",
```

Comment:
That's good for the default container profile, but I wonder about the userns profile:
- if we use userns, we're already shifting the uids, so using root should be fine, since it's not the real root
- how is the string "nobody" converted to an uid? Without userns, I would think it uses `/etc/passwd` of the host, but that mapping should not be relevant in a different userns.

Also, on my Fedora laptop, nobody is uid 99:
```
$ grep nobody /etc/passwd
nobody:x:99:99:Nobody:/:/sbin/nologin
nfsnobody:x:65534:65534:Anonymous NFS User:/var/lib/nfs:/sbin/nologin
```

I would not want nobody to be mapped to 65534 because of the special meaning it has with in userns (see `man user_namespaces`) but in some distro, nobody could be mapped to 65534...